Makefile

```diff
@@ -146,24 +146,26 @@ SPECTRAL ?= ${LOCAL_BIN_PATH}/spectral
 NPM ?= "$(shell which npm)"
 specinstall:
 ifeq (, $(shell which ${NPM} 2> /dev/null))
-	@echo "npm is not available please install it to be able to install spectral"
-	exit 1
+       @echo "npm is not available please install it to be able to install spectral"
+       exit 1
 endif
-ifeq (, $(shell which ${LOCAL_BIN_PATH}/spectral 2> /dev/null))
+ifeq (, $(shell which ${LOCAL_BIN_PATH}/rhoasapi 2> /dev/null))
```

Comment:
Could you give more context on what is `rhoasapi`, the difference/s with `spectral` and why has it been changed?

Reply:
`rhoaspi` is essentially spectral binary thats been wrapped up in the rhoas guidelines package

Reply:
> `rhoaspi` is essentially spectral binary thats been wrapped up in the rhoas guidelines package

Does the wrapping assumes that `spectral` cli is already installed?
I am getting 
```
bin/rhoasapi: line 5: spectral: command not found
npm WARN using --force I sure hope you know what you are doing.
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /usr/local/lib/node_modules/.staging
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/.staging'
npm ERR!  [OperationalError: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/.staging'] {
npm ERR!   cause: [Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/.staging'] {
npm ERR!     errno: -13,
npm ERR!     code: 'EACCES',
npm ERR!     syscall: 'mkdir',
npm ERR!     path: '/usr/local/lib/node_modules/.staging'
npm ERR!   },
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/usr/local/lib/node_modules/.staging',
npm ERR!   parent: '@rollup/pluginutils'
npm ERR! }
npm ERR! 
```

when I run the change locally.

Reply:
no it doesn't. they are both bundled together. So that is the output when you run `make openapi/spec/validate`?

Reply:
Yes. That's what I got when I run `make openapi/spec/validate`

Reply:
I've pushed some new changes, only a new directory is now created (this just takes the place of the old `spectral-installation` dir.  This is done to facilitate the easy clean up of existing spectral and RHOAS Guidelines which may be on dev machines.

running `make openapi/spec/validate` should now clean up, install new versions and lint.